lib/OpenQA/BuildResults.pm

```diff
@@ -146,6 +146,13 @@ sub compute_build_results ($group, $limit, $time_limit_days, $tags, $subgroup_fi
         order_by => {-desc => $sort_column},
         rows => $row_limit
     );
+    # SELECT VERSION, BUILD, MAX(id) as newest_job, MIN(id) as oldest_job, nj.t_created, oj.t_created FROM jobs
+    # WHERE $search_filter
+    # GROUP BY VERSION, BUILD
+    # ORDER BY $sort_column DESC
+    # LEFT JOIN jobs as nj ON nj.id = newest_job
+    # LEFT JOIN jobs as oj ON oj.id = oldest_job
+    # LIMIT $row_limit
```

Comment:
I also think that SQL is easier to read then the DBIx query. However, I would still avoid duplicated and commented-out code. This SQL is also a bit of a simplification and I think the order in which `WHERE`, `LEFT JOIN` and `ORDER BY` occur here is actually wrong.